packages/ember-data/tests/unit/record_array_test.js

```diff
@@ -49,6 +49,11 @@ test("stops updating when destroyed", function() {
   expect(3);
   var store = createStore();
 
+  // TODO remove once
+  // https://github.com/emberjs/ember.js/commit/c3f13e85a62069295965dd49ca487fe6ddba1188
+  // is on the release branch
+  var emptyLength = Ember.meta(store).descs ? undefined : 0;
```

Comment:
`length` on an ArrayProxy is a computed property and they are no longer removed when an object is destroy now that they are not stored on the meta object.